cmd/satellite/main.go

```diff
@@ -301,6 +315,45 @@ func cmdQDiag(cmd *cobra.Command, args []string) (err error) {
 	return w.Flush()
 }
 
+func cmdGracefulExit(cmd *cobra.Command, args []string) (err error) {
+	ctx, _ := process.Ctx(cmd)
+
+	layout := "2006-01-02"
+	start, err := time.Parse(layout, args[0])
+	if err != nil {
+		return errs.New("Invalid date format. Please use YYYY-MM-DD")
+	}
+	end, err := time.Parse(layout, args[1])
+	if err != nil {
+		return errs.New("Invalid date format. Please use YYYY-MM-DD")
+	}
+
+	//Adding one day to properly account for the entire end day
+	end = end.AddDate(0, 0, 1)
+
+	// Ensure that start date is not after end date
+	if start.After(end) {
+		return errs.New("Invalid time period (%v) - (%v)", start, end)
+	}
+
+	// send output to stdout
+	if gracefulExitCfg.Output == "" {
+		return generateGracefulExitCSV(ctx, gracefulExitCfg.Completed, start, end, os.Stdout)
+	}
```

Comment:
nit - missing space and inconsistent capitalization

Reply:
Fixed